src/main/kotlin/org/rust/ide/presentation/Ty.kt

```diff
@@ -9,12 +9,21 @@ import org.rust.lang.core.types.ty.*
 
 
 val Ty.shortPresentableText: String get() = render(this, level = 3)
+val Ty.insertionSafeText: String
+    get() = render(this, level = 3, unknown = "_", anonymous = "_", integer = "_", float = "_")
```

Comment:
Hm, we probably should not use `level` when rendering types for insertion in code?

Reply:
Okay